x/ecocredit/core/features/msg_bridge_receive.feature

```diff
@@ -129,7 +129,7 @@ Feature: MsgBridgeReceive
     }
     """
     When the message is validated
-    Then expect the error "invalid jurisdiction: foo, expected format <country-code>[-<region-code>[ <postal-code>]]: parse error: invalid request"
+    Then expect the error "invalid jurisdiction: foo, expected format <country-code>[-<region-code>[ <postal-code>]]: invalid request: invalid request"
```

Comment:
In line with my comments above, I think the previous error is the desired error message here, which would include the parse error from the utility function and the invalid request error from basic message validation.